makefile.shade

```diff
@@ -293,66 +298,13 @@ var buildTarget = "compile"
 
 #cache-coherence
     @{
-        if (Directory.Exists(dropsShare))
+        if (string.IsNullOrEmpty(coherencePath))
         {
-            var coherenceShare = Path.Combine(dropsShare, "Coherence", buildBranch);
-            var latestBuild = Directory.EnumerateDirectories(coherenceShare)
-                .Select(d => Path.GetFileName(d))
-                .Where(d => { int _; return int.TryParse(d, out _); })
-                .OrderByDescending(d => d)
-                .First();
-            var latestBuildShare = Path.Combine(coherenceShare, latestBuild, "build");
-            var coherenceVersionFilePath = Path.Combine(coherenceCacheDir, CoherenceCacheVersionFileName);
-            var universeCommitsFileTarget = Path.Combine(coherenceCacheDir, UniverseCommitsFileName);
```

Comment:
This directory is no longer produced (packages are split across multiple dirs). What we do produce is a directory with  expanded packages (with the right casing). These are copied over as build artifacts so we no longer need to do this caching explicitly.